src/main/webapp/join-room.js

```diff
@@ -13,11 +13,51 @@ const createRoom = async (ev) => {
       },
     });
   
-    // Sends user to room
+    // Sends user to room.
     window.location.href = 'lobby.html?roomId=' + room.roomId;
 }
 
+const joinRoom = async (ev) => {
+  ev.preventDefault();
+
+  // Get url from user input.
+  let url = document.getElementById('room-url-id').value;
+  let room = parseRoomId(url);
+  
+  // Put room object.
+  fetch("/room", {
+    method: "PUT",
+    body: JSON.stringify(room),
+    headers: {
+      "Content-Type": "application/json",
+    },
+  });
+
+  // Sends user to entered room.
+  window.location.href = url;
+}
+
+function parseRoomId(url) {
```

Comment:
Can you add some tests for this function in this PR? Other than that, everything looks good